protocol/x/subaccounts/keeper/transfer.go

```diff
@@ -262,8 +264,17 @@ func (k Keeper) DistributeFees(
 	// Remaining amount goes to the fee collector
 	feeCollectorShare := new(big.Int).Sub(quantums, marketMapperShare)
 
+	// Emit a metric for the amount of fees transferred to the market mapper
+	labels := []metrics.Label{
+		metrics.GetLabelForIntValue(metrics.MarketId, int(perpetual.Params.MarketId)),
+	}
+	metrics.AddSampleWithLabels(
+		metrics.MarketMapperRevenue,
+		metrics.GetMetricValueFromBigInt(marketMapperShare),
+		labels...,
+	)
```

Comment:
What do you plan on doing with this metric? It's emitted per block, but gets sampled, does that mean we get some approx. sum over the blocks or approx. average?

Reply:
Hmm i hope these metrics don't get sampled. At least the IncrCounter and AddSample ones.
How do you know these are sampled?

Reply:
I see we use the same metric to count liquidations and deleveraging: https://github.com/dydxprotocol/v4-chain/blob/dfd4f6798265101bbd52a17c92da3d3a1a87652c/protocol/x/clob/keeper/deleveraging.go#L111

Reply:
Asked around and these metrics are not sampled by default

Reply:
`AddSample` does do summarization of metrics, but the summary does track the `sum` which is what we need. Can you update the metric name to indicate this, e.g. similar to the deleverage metric with the `distribution` suffix?